tools/download.sh

```diff
@@ -25,21 +33,22 @@ case "$OSTYPE" in
                     echo "curl not found, installing..."
                     sudo yum install -y curl
                 fi
-
-                package=powershell-6.0.0_alpha.18-1.el7.centos.x86_64.rpm
+                get_currentpshversion
+                package=powershell-$release-1.el7.centos.x86_64.rpm
                 ;;
             ubuntu)
                 if ! hash curl 2>/dev/null; then
                     echo "curl not found, installing..."
                     sudo apt-get install -y curl
                 fi
+                get_currentpshversion
 
                 case "$VERSION_ID" in
                     14.04)
-                        package=powershell_6.0.0-alpha.18-1ubuntu1.14.04.1_amd64.deb
```

Comment:
I think it should just install the latest version from the repo.  Eventually, we will have a non-pre-release version, and I think then we have a decision to make, but currently, if it's in the repo, it's ready.